bc_obps/registration/api/v2/_operators/operator_id.py

```diff
@@ -28,19 +25,3 @@
 @handle_http_errors()
 def get_operator(request: HttpRequest, operator_id: UUID) -> Tuple[Literal[200], Operator]:
     return 200, OperatorDataAccessService.get_operator_by_id(operator_id)
-
-
-@router.put(
```

Comment:
Operators don't get approved in v2 anymore (should have thought of that while I was doing the v1/v2 refactor)